src/messages/en.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ChartTableFilterHead": {
+    "searchAll": "Search all entries",
+    "clearTooltip": "Clear all filters",
+    "collapseTooltipExpand": "Expand filters",
+    "collapseTooltipCollapse": "Collapse filters"
+  },
+  "ChartTableFilterBody": {
+    "search": "Search"
+  },
+  "ChartTableWrapper": {
+    "CHART": "Bar Chart",
+    "TABLE": "Table"
+  },
+  "Table": {
+    "rowsPerPageText": "Rows per page:",
+    "rangeSeparatorText": "of",
+    "selectAllRowsItemText": "All"
+  },
+  "LocaleSwitcher": {
+    "locale": "{locale, select, de {🇩🇪 Deutsch} en {🇬🇧 English} other {Unknown}}"
+  },
+  "NotFound": {
+    "title": "Oops! You're lost.",
+    "subtitle": "The page you are looking for was not found.",
+    "returnBtn": "Back to Home"
```

Comment:
Change this to `Return Home`, or `Return to Homepage` or something like that. `Back to Home` sounds weird.